docs/design/network-discovery/port-FSM.puml

```diff
@@ -39,9 +39,11 @@ state OPERATIONAL {
 
     UP --> DOWN : port-down
     UP : enter / emit port-down into report
-    UP : enter / issue addWatch
+    UP : enter / issue addWatch if discovery enabled
     UP : poll-discovery / emit discovery
     UP : poll-fail / emit fail
+    UP : enable-discovery / issue addWatch
+    UP : disable-discovery / issue delWatch
```

Comment:
Mention "set enabled = false" here. As far as I understood it is done here.

Reply:
Added a readme. Port Fsm should read enabled/disabled flag from DB on "UP : enter" so we don't need to save flag manually.